### PR TITLE
SYNPY-727

### PR DESCRIPTION
**Pull request by kimyen**

### CODE REVIEW

File: tests/integration/integration_test.py [hunk 1]
```diff
@@ -318,28 +327,29 @@ def test_annotations():
...
     syn.setAnnotations(entity, annote)
     
     # Check it again
     annotation = syn.getAnnotations(entity)
-    assert annotation['primes'] == [2,3,5,7,11,13,17,19,23,29]
+    assert annotation['primes'] == [2, 3, 5, 7, 11, 13, 17, 19, 23, 29]
     assert annotation['phat_numbers'] == [1234.5678, 8888.3333, 1212.3434, 6677.8899]
     assert annotation['goobers'] == ['chris', 'jen', 'jane']
-    assert annotation['present_time'][0].strftime('%Y-%m-%d %H:%M:%S') == annote['present_time'].strftime('%Y-%m-%d %H:%M:%S')
+    assert annotation['present_time'][0].strftime('%Y-%m-%d %H:%M:%S') == \
```
[the remark this answers is not in the document]
Comment (kimyen) [reply tests/integration/integration_test.py]: This pull request is getting very long. I will address all `assert` in a separate pull request.

File: tests/integration/integration_test_Entity.py [hunk 2]
```diff
@@ -397,12 +399,12 @@ def test_synapseStore_flag():
     bogus = syn.get(bogus, downloadFile=False)
     assert bogus.name == 'Totally bogus data'
     assert bogus.path == path, "Path: %s\nExpected: %s" % (bogus.path, path)
-    assert bogus.synapseStore == False
+    assert_false(bogus.synapseStore)
 
     # Make sure the test runs on Windows and other OS's
-    if path[0].isalpha() and path[1]==':':
+    if path[0].isalpha() and path[1] == ':':
```
[the remark this answers is not in the document]
Comment (kimyen) [reply tests/integration/integration_test_Entity.py]: I will address this in the next clean up.

File: tests/unit/unit_test_credentials/unit_test_cred_provider.py [hunk 3]
```diff
@@ -61,34 +60,34 @@ def test_get_credentials__multiple_providers(self):
...
             self.provider.get_synapse_credentials(syn, self.user_login_args)
 
             mock_get_auth_info.assert_called_once_with(syn, self.user_login_args)
             mock_create_synapse_credentials.assert_called_once_with(syn, *auth_info)
 
-
     def test_create_synapse_credential__username_is_None(self):
         # shouldn't matter what the other fields are if username is None
         cred = self.provider._create_synapse_credential(syn, None, self.password, self.api_key)
         assert_is_none(cred)
```
[the remark this answers is not in the document]
Comment (kimyen) [reply tests/unit/unit_test_credentials/unit_test_cred_provider.py]: Module's methods and classes are separated by 2 lines. Inner methods and classes (of a module's class) are separated by 1 line.

File: tests/unit/unit_test_retry.py [hunk 4]
```diff
@@ -45,15 +43,17 @@ def theCharm(x):
     response.status_code.__eq__.side_effect = lambda x: x == 500
     response.headers.__contains__.reset_mock()
     response.headers.__contains__.side_effect = lambda x: x == 'content-type'
-    response.headers.get.side_effect = lambda x,default_value: "application/json" if x == 'content-type' else None
+    response.headers.get.side_effect = lambda x, default_value: "application/json" if x == 'content-type' else None
```
[the remark this answers is not in the document]
Comment (kimyen) [reply tests/unit/unit_test_retry.py]: Will address this in the next pull request.

File: tests/unit/unit_test_credentials/unit_test_cred_provider.py (hunk 3 above)
[the remark this answers is not in the document]
Comment (kimyen) [reply tests/unit/unit_test_credentials/unit_test_cred_provider.py]: https://www.python.org/dev/peps/pep-0008/?#blank-lines

File: tests/integration/integration_test.py [hunk 5]
```diff
@@ -196,17 +203,15 @@ def test_uploadFile_given_dictionary():
...
 
 def test_uploadFileEntity():
     # Create a FileEntity
     # Dictionaries default to FileEntity as a type
     fname = utils.make_bogus_data_file()
     schedule_for_cleanup(fname)
-    entity = {'name'        : 'fooUploadFileEntity', \
-              'description' : 'A test file entity', \
-              'parentId'    : project['id']}
+    entity = {'name': 'fooUploadFileEntity', 'description': 'A test file entity', 'parentId': project['id']}
```
[the remark this answers is not in the document]
Comment (kimyen) [reply tests/integration/integration_test.py]: I agree. This is a style that is not included in `inspect code`. I do want to keep it consistent and easy to read. In future clean up, I will keep the key/value pairs on separate lines.

File: tests/unit/unit_test_credentials/unit_test_cred_provider.py (hunk 3 above)
[the remark this answers is not in the document]
Comment (zimingd) [reply tests/unit/unit_test_credentials/unit_test_cred_provider.py]: Sorry about that. I misunderstood the convention. Thanks for the correction!

**Comment by zimingd** [comment]
Kim said that this pull request was about correcting warnings from pylint and that the pull request is already rather long. I'll merge this and wait for the next pull request to address the requested changes above

**Pull request merged**